wgsl/index.bs

```diff
@@ -728,36 +728,38 @@ type.
 </pre>
 * type must be `f32`, `i32` or `u32`
 
-### Read-only Storage Texture Types ### {#texture-ro}
+### Storage Texture Types ### {#texture-storage}
 
 A <dfn noexport>read-only texture</dfn> is a texture from which individual texels
-may be read, without the use of a sampler.
-See [[#texture-builtin-functions]].
+may be read, without the use of a sampler. A <dfn noexport>write-only texture</dfn>
+is a texture to which individual texels may be written. See [[#texture-builtin-functions]].
+The `read-only` and `write-only` is specified through a decorator when creating
```

Comment:
It will apply any time you state the type, which we'll also have for function parameters.  So I suggest we generalize by replacing 

> The `read-only` and `write-only` is specified through a decorator when creating the texture variable. 

with:

> A storage texture type must always have one `access` attribute. The access attribute parameter must be either `read` to specify a read-only texture,  or `write` to specify a write-only texture.

This new phrasing I think makes it clearer that the attribute is part of the type.

Reply:
done.